ui/app/AppLayouts/Chat/panels/communities/CommunityProfilePopupOverviewPanel.qml

```diff
@@ -73,7 +73,7 @@ Column {
 
     StatusListItem {
         anchors.horizontalCenter: parent.horizontalCenter
-        visible: !root.community.amISectionAdmin
+        visible: root.community.memberRole !== Constants.memberRole.owner && root.community.memberRole !== Constants.memberRole.admin
```

Comment:
I think it was decided that Admins can leave, just not the Owner, since the Owner has the private key (most of the time)